### PR TITLE
Add colors to text of method

### PR DESCRIPTION
**Pull request by sak96**
Fixes: #32 

Note:

- The texted is colored instead of background.
- color used might not be good selection

# source for color

- [swagger light theme](https://github.com/swagger-api/swagger-ui/blob/b0ab56187915677d99b3dc28120a9286aacdedc2/src/style/_variables.scss#L46)
- [swagger dark theme](https://github.com/Amoenus/SwaggerDark/blob/b6fa2536da4e60b3fa15fa5e62eaac668dbc2a21/SwaggerDark.css#L471) 

## Dark mode

image

## Light mode

image

### CODE REVIEW

File: mitm_proxy/src/requests.rs [hunk 1]
```diff
@@ -109,7 +112,20 @@ impl RequestInfo {
...
+                Method::GET => Color32::GREEN,
+                Method::POST => Color32::BLUE,
+                Method::PUT => Color32::LIGHT_BLUE,
+                Method::DELETE => Color32::RED,
+                Method::HEAD => Color32::YELLOW,
+                Method::TRACE => Color32::BROWN,
+                Method::CONNECT => Color32::GOLD,
+                Method::PATCH => Color32::DARK_BLUE,
+                _ => Color32::DARK_GRAY,
+            };
```
Comment (emanuele-em) [inline mitm_proxy/src/requests.rs:127]: probably a better solution should to use a separate function to manage colors
`let color = get_color(method)`

**Review by emanuele-em — COMMENTED** [review]
thank you for your contribution!
Only some recommendation 🦀

File: mitm_proxy/src/requests.rs [hunk 2]
```diff
@@ -109,7 +112,20 @@ impl RequestInfo {
         });
 
         row.col(|ui| {
-            ui.label(req.method().to_string());
```
[the remark this answers is not in the document]
Comment (sak96) [reply mitm_proxy/src/requests.rs:112]: sure.

But do you have some idea on what colors look good ?

[the remark this answers is not in the document]
Comment (emanuele-em) [reply mitm_proxy/src/requests.rs:112]: [here](https://materialui.co/colors/) some colors, look at first lines, 50-200

they should be less shock

**Review by emanuele-em — APPROVED** [review]
Looks good!!!!

**Pull request merged**